assets/js/googlesitekit-idea-hub-post-list-notice.js

```diff
@@ -35,15 +35,12 @@ domReady( () => {
 	if ( ! notice ) {
 		return;
 	}
-	const type = notice.id.replace( 'googlesitekit-notice-idea-hub_', '' );
+	const type = notice.id.replace( 'googlesitekit-notice-', '' );
+	const expiresInSeconds = type === 'idea-hub_new-ideas' ? WEEK_IN_SECONDS : 0;
 
-	// Button pops up only after the timeout passes
-	setTimeout( () => {
-		const button = notice.querySelector( '.notice-dismiss' );
-		if ( ! button ) {
-			return;
+	notice.addEventListener( 'click', ( event ) => {
+		if ( 'notice-dismiss' === event.target.className ) {
```

Comment:
Let's make it conditional base on the presence of the class, rather than it _only having that class_ 🙂 
```suggestion
		if ( event.target.classList.contains( 'notice-dismiss' ) ) {
```